roles/ceph-osd-bcache/tasks/main.yml

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
This looks like it shares a lot of code with `roles/ceph-osd/`  Is there no way to stash the bcache specific tasks into an include file, with a conditional? I'm just wary of drift between the two files as we fix things or change things that are common across both.